app/src/main/java/com/hedvig/app/feature/home/model/HomeItemsBuilder.kt

```diff
@@ -0,0 +1,163 @@
+package com.hedvig.app.feature.home.model
+
+import androidx.compose.ui.unit.dp
+import arrow.core.NonEmptyList
+import com.hedvig.android.owldroid.graphql.HomeQuery
+import com.hedvig.android.owldroid.type.PayinMethodStatus
+import com.hedvig.app.R
+import com.hedvig.app.feature.claims.ui.commonclaim.CommonClaimsData
+import com.hedvig.app.feature.claims.ui.commonclaim.EmergencyData
+import com.hedvig.app.feature.home.ui.claimstatus.data.ClaimStatusCardUiState
+import com.hedvig.app.util.featureflags.Feature
+import com.hedvig.app.util.featureflags.FeatureManager
+
+@OptIn(ExperimentalStdlibApi::class)
+class HomeItemsBuilder(
+    private val featureManager: FeatureManager
+) {
```

Comment:
A great idea! This did end up being quite a lot, with a lot of mixed cases. This feels like it will make it easier to work with going forward!